### ISSUE TITLE
Nullable Property Types don't work

### ISSUE THREAD

**Issue by garywebb** [garywebb/CherryPicker#3]
A StructureMap.StructureMapConfigurationException exception is thrown with message:

> Invalid valueOrInstance '23/08/2018 00:00:00' for parameter dateTime of type Nullable<DateTime<xxx>> 

The exception is thrown when trying to set a default or override for a Property with a Nullable type.

```c#
public class Test {
    public int? Test1 { get; set; }
}

//This doesn't work
new TestDataContainer().For<Test>(x => x
    .Default(t => t.Test1).To(3));
```

**Comment by garywebb** [comment]
This is a bug which will be fixed in v0.2.1.

I haven't found a workaround, so unfortunately, users will have to upgrade to the latest version.

The problem is in StructureMap (which is used to build the data objects). It calls Convert.ChangeType to try and convert the value passed in to the property's type, but the ChangeType method can't handle conversions from non-nullables to nullables.

**Comment by garywebb** [comment]
Bug has been fixed
